src/cmake/openvino.cmake

```diff
@@ -24,6 +24,7 @@ add_library(${TARGET_NAME}
     $<TARGET_OBJECTS:frontend_common_obj>
     $<TARGET_OBJECTS:inference_engine_obj>
     $<TARGET_OBJECTS:inference_engine_obj_version>
+    $<TARGET_OBJECTS:openvino_proxy_plugin_obj>
```

Comment:
do we need an option to disable proxy plugin? can be required to optimize binary size.